Core/Src/torque.c

```diff
@@ -74,4 +145,23 @@ void vCalcTorque(void* pv_params)
 		dti_set_torque(torque);
 	}
 }
- 
\ No newline at end of file
+
+void increase_torque_limit()
```

Comment:
nit: maybe move these up in the file, I like the task to be at the very bottom so it is easy to find